### PR TITLE
Adding consolidated muting docs

### PR DESCRIPTION
**Pull request by esthermmoturi**

### CODE REVIEW

**Comment by michaelkohn** [comment]
I'm going to add a number of comments below, but wanted to first make sure that what I'm seeing is what you are expecting me to see: Muting content added to Features and Tutorials. 

Screenshot 2022-12-13 at 5 05 50 PM



### Some initial context
1. The "Features" section should be somewhat brief, high level, and understandable to a non-technical audience. 
2. The "Tutorials" section is for step-by-step instructions on _configuring_ a specific pre-defined scenario that will highlight a specific feature, so the target audience would be an app developer.
3. The "Quick Guides" section get's more technical than the "Features" section and highlights key technical settings, but doesn't go so far as identifying a specific scenario to configure and step-by-step instructions like "Tutorials" does. 

### Recommended approach
I would recommend focusing on (and just having a single PR for) the content for the "Features" section first. Let's get that published and then we can work on content for Quick Guides and then Tutorials.

### Some recommendations for the Muting content for the "Features" section:
1. It should only be one page
2. It should be understandable by non-technical folks
3. Make sure the content is written in narrative so that it's readable. It shouldn't be just a bunch of headings and bullet points.
4. Try to focus on muting as a concept, not about a specific implementation of it. Remember that there's really no such thing as a "Mute" form, I could configure a pregnancy registration form to mute a person if it made sense in my project. 
5. Have a look at [UHC Mode](https://docs.communityhealthtoolkit.org/apps/features/uhc-mode/) and [Tasks](https://docs.communityhealthtoolkit.org/apps/features/tasks/) content for inspiration regarding tone, level of detail, and format.

### Potential structure for the content: 
1. Briefly explain what "Muting" does... and don't use the word "mute" when describing what it does ;-) 
2. Give some examples of when you might want to mute a person or place. You already have a good list of _"Scenarios where muting is applicable"_. Turn two or three of those bullet points into a readable paragraph.
3. Briefly describe _how_ it's done (ie submitting a form)
4. Show some screenshots of what it looks like when a person or place is muted
6. Talk a bit about what you can configure (high level - ie through configuration you can specify which forms mute, which forms unmute, etc...) but don't describe _how_ to configure. Use [this](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#configuration-4) as a guide.
7. You could include the table that differentiates death reporting from muting
8. In "Related Content", you could link to [this](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/#muting)

**Comment by esthermmoturi** [comment]
@michaelkohn I left the name family on most parts of the document rather than household to stay consistent with the screenshots. Feel free to have a look and point out any changes that are needed.

File: content/en/apps/features/muting/_index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Muting vs Death Reporting
+
+A project may support both death reporting and muting - they are not mutually exclusive. Death reporting moves the deceased person to a different part of the family members list and does not allow actions. Muting keeps the person in the family members list, allows actions and disables schedules.
+
+|   Death reporting                                                                                        | Muting                                                                                         |
+| -------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| <p></p><ul><li>Permanent state</li></ul>                                                                 | <ul><li>Temporary state</li></ul>                                                              |
```
Comment (michaelkohn) [inline content/en/apps/features/muting/_index.md:53]: The bullet points do not line up properly for all rows except the last one. You can see that the death reporting bullet points are a bit lower than the ones for Muting. 
 
Screenshot 2023-01-13 at 11 22 21 PM

File: content/en/apps/features/muting/_index.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Some changes are observed in the UI when a family or a person is muted to make the muted status clear and easily recognizable. The person's or family's icon turns grey  and a status of “Muted” displays on the second line below the person's or family's name. Any actions that were previously available on the family or person remain available on the profile.
+
+{{< figure src="muted-family-appearance-on-list.png" link="muted-family-appearance-on-list.png" class="col-3 col-lg-6" >}}
+
+For the cases where families are muted, each of the family members in the people card are also styled as muted, (grey icons and “Muted” labels).A user may add a new person to a muted household. If they do, and choose to leave the family muted, the new person will be automatically muted once created. When a family is unmuted, all individuals in the family will be unmuted at the same time. 
+
+Unmuting an individual person in a muted family works essentially the same way, the action unmutes the entire family because unmuted individuals  are not allowed underneath a muted household.
+
+### Configuration
```
Comment (michaelkohn) [inline content/en/apps/features/muting/_index.md:43]:
```suggestion
### Configurability
```

File: content/en/apps/features/muting/_index.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Through [configuration]({{< relref "apps/reference/app-settings/transitions" >}}), you can specify which form mutes and unmutes, form availability, form questions and options, manager verification and labels used for display and warning messages.
```
Comment (michaelkohn) [inline content/en/apps/features/muting/_index.md:45]:
```suggestion
While it's most common to have dedicated mute and unmute forms, any form can be set up to mute or unmute a Contact. It's also possible to set up verification flows for muting whereby mute/unmute requires a Supervisor to verify before the mute/unmute happens. And like the rest of the CHT, all text is customizable.  

```

File: content/en/apps/features/muting/_index.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+---
+title: Muting
+weight: 10
```
Comment (michaelkohn) [inline content/en/apps/features/muting/_index.md:3]: I would actually put this after UHC Mode, before App Management

**Review by michaelkohn — CHANGES_REQUESTED** [review]

File: content/en/apps/features/muting/_index.md (hunk 1 above)
Comment (esthermmoturi) [reply content/en/apps/features/muting/_index.md:52]: Thanks for noting this, I have corrected the alignment

File: content/en/apps/features/muting/_index.md (hunk 4 above)
Comment (esthermmoturi) [reply content/en/apps/features/muting/_index.md]: Done

**Comment by esthermmoturi** [comment]
Closing this PR as it was merged on https://github.com/medic/cht-docs/pull/945

**Pull request closed without merging**